legal-api/src/legal_api/core/filing.py

```diff
@@ -45,7 +46,7 @@ def __init__(self):
         self._raw: Optional[Dict] = None
         self._completion_date: datetime
         self._filing_date: datetime
-        self._filing_type: str
+        self._filing_type: str = ''
```

Comment:
Looking feedback from @lekshmimallika-aot and @kialj876 as well.
Should we do
```
Optional[str]
```
or
```
str=''
```
I'm thinking we should note something about our coding standards here as we provide no guidance on typing yet.

Reply:
Optional looks good to me